webpack.config.js

```diff
@@ -107,6 +107,12 @@ module.exports = function(env) {
                     to: "charting_library"
                 }
             ]
+        }),
+        new webpack.ProvidePlugin({
```

Comment:
can you elaborate why those two new plugins are necessary for gdex? In particular, which lines of code need this?

Reply:
This is mainly because webpack 5 no longer automatically packs processes and buffers. The code location is:
https://github.com/bitshares/bitshares-ui/blob/develop/app/components/DepositWithdraw/gdex/GdexGateway.jsx#L18

Reply:
Thanks!